lib/miq_automation_engine/engine/miq_ae_object.rb

```diff
@@ -279,10 +279,18 @@ def process_args_attribute(args, args_key)
     end
 
     def load_array_objects_from_string(objects_str)
-      objects_str.split(',').collect do |o|
-        klass, str_value = o.split(CLASS_SEPARATOR)
-        value = MiqAeObject.convert_value_based_on_datatype(str_value, klass)
-        value if value.kind_of?(MiqAeMethodService::MiqAeServiceModelBase)
+      objects_str.split(',').collect do |element|
+        if element.include?(CLASS_SEPARATOR)
+          klass, str_value = element.split(CLASS_SEPARATOR)
+          value = MiqAeObject.convert_value_based_on_datatype(str_value, klass)
```

Comment:
@gmcculloug - I ran into an issue with `klass` if `strip` is not called here.  

when a string is passed in "integer::1, integer::3, integer::5" klass is set to " integer" and doesn't resolve correctly in `convert_value_based_on_datatype` https://github.com/gmcculloug/manageiq/blob/b07a184a9c7cee349fac2725d93dc7afd6485a72/lib/miq_automation_engine/engine/miq_ae_object.rb#L532